plugins/storage/pyparts/bcache.py

```diff
@@ -1,11 +1,39 @@
 import re
 
 from core.cli_helpers import CLIHelper
-from core.plugins.storage.bcache import BcacheChecksBase
+from core.plugins.storage.bcache import BcacheBase, BcacheChecksBase
+from core.plugins.juju import JujuChecksBase
+from core.issues import issue_types, issue_utils
 
 YAML_PRIORITY = 3
 
 
+class BcacheCharmChecks(BcacheBase, JujuChecksBase):
+
```

Comment:
So it is kindof personal preference but I feel it makes for clearer code to avoid multiple inheritance where possible and so I would suggest leaving this as inheriting from BcacheChecksBase and then just instantiate a JujuChecksBase object inside the bcache_tuning_unit method.

Reply:
Agree that's a better way. Modified as suggested.